docs/testplan/token-payment-processor-testplan.md

```diff
@@ -0,0 +1,49 @@
+# Token Payment Processor Testplan
+
+&nbsp;
+
+## Background
+This testplan is going to cover the token payment processor, it will go over must haves and additional features that we can add on later. 
+
+UI relating to token payment processing can be seen on [Billing Page](https://www.figma.com/file/HlmasFJNHxs2lzGerq3WYH/Satellite-GUI-Public?node-id=11080%3A68109) figma design.
+
+&nbsp;
+
+&nbsp;
+
+<google-sheets-html-origin><!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}-->
+
+Test Scenario | Test Case | Description | Comments
+-- | -- | -- | --
+Transaction | Block Chain Reorg | If there is a naturally occurring chain reorg then past transactions in the old chain are deactivated due to a chain reorganization (this should be visible in UI, pending to cancelled) | Must have early on
+  | Transaction Override | If a user overrides the first transaction with a higher transaction fee, then the token payment processor has to replace the transaction hash it is watching (this should be visible in UI) | Must have early on
```

Comment:
Can we split "Must have early on" into 2 groups? Let's call it "Must have for Alpha Version" and "Must have for Beta Version". The first Alpha Version might not be able to handle pending transactions.